content/en/blog/_posts/2019-05-22-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

```diff
@@ -0,0 +1,178 @@
+
+---
+layout: blog
+title: "OPA Gatekeeper: Policy and Governance for Kubernetes"
+date: 2019-05-22
+slug: OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes 
+---
+
+**Authors:** Rita Zhang (Microsoft)
+
+In this post, we will walkthrough the goals, history, and current state of the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) project, and how you can leverage it to enforce policies and strengthen governance in your Kubernetes environment. 
+
+You can also check out recordings from the following Kubecon EU 2019 sessions:
+* [Intro: Open Policy Agent Gatekeeper](https://sched.co/MPiM)
+* [Deep Dive: Open Policy Agent](https://sched.co/MPk0)
+
+## Motivations 
+
+If your organization has been operating Kubernetes, you probably have been looking for ways to control what end-users can do on the cluster and ways to ensure the clusters are always in compliance with company policies. These policies may be there to meet governance and legal requirements or to enforce best practices and organizational conventions. With Kubernetes, how do we ensure compliance without sacrificing development agility and operation independence? 
+
+Kubernetes allows decoupling policy decisions from the API server by means of [admission controller webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) to intercept admission requests before they are persisted as objects in Kubernetes. [Gatekeeper](https://github.com/open-policy-agent/gatekeeper) was created to enable users to customize admission control via configuration, not code and to bring awareness of the cluster’s state, not just the single object under evaluation at admission time. Gatekeeper is a customizable admission webhook for Kubernetes that enforces policies executed by the [Open Policy Agent (OPA)](https://www.openpolicyagent.org), a policy engine for Cloud Native environments hosted by CNCF.
+
```

Comment:
A few concrete policy examples would be useful here.  Image-repository safety and ingress-conflicts are my 2 goto examples because (i) they're real policies that people will put in place immediately and (ii) they illustrate stateless and stateful policies.